SampleApps/FRExample/FRExample/AppDelegate.swift

```diff
@@ -27,6 +33,17 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     
     func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey : Any] = [:]) -> Bool {
         
+        #if canImport(FRFacebookSignIn)
```

Comment:
Is this the wrong way around? Shouldn't it be:

#if canImport(FRFacebookSignIn)
        if FacebookSignInHandler.handle(app, url, options) {
            return true
        }
        #endif
        #if canImport(FRGoogleSignIn)
        
        if GoogleSignInHandler.handle(app, url, options) {
            return true
        }
        #endif

Reply:
Sorry about the formatting but it really doesn't like me pasting the code :)

Reply:
@george-bafaloukas-forgerock Thanks for pointing this out.

Reply:
Updated